apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/InpatientClaimTransformerV2.java

```diff
@@ -342,7 +343,7 @@ private static ExplanationOfBenefit transformClaim(InpatientClaim claimGroup) {
               c ->
```

Comment:
why not use the `TransformerUtilsV2.mapHcpcs()` function that the other transformers are using? https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/OutpatientClaimTransformerV2.java#L286-L291

Reply:
That is on me actually. I asked for a smaller change because at first glance it appeared to me that there were some differences that led to this being done this way. Having looked at it more closely now I think @cbrunefearless 's original fix that did what you suggest is valid and preferable so I'm ok with going back to that.